src/assemblyItemsBrowser.js

```diff
@@ -108,11 +109,17 @@ module.exports = React.createClass({
 		return ret
 	},
 
-	resolveAddress: function(address)
+	resolveAddress: function(address, props)
 	{
 		if (!this.state.codes[address])
 		{
-			var hexCode = web3.eth.getCode(address)	
+			console.log("loading new code from web3 " + address)
+			var hexCode
+			if (this.state.currentSelected === -1 && props.transaction.to === null) // contract creation
+				hexCode = props.transaction.input
+			else
+				hexCode = web3.eth.getCode(address)
+			
 			var code = codeUtils.nameOpCodes(new Buffer(hexCode.substring(2), 'hex'))
 			this.state.codes[address] = code[0]
```

Comment:
This property is only used to cache the code (in case we call another contract several times).
I see what you mean but this cache is here to store the code representation of an address, and for a creation transaction the code is the creation code.
The name is confusing, i will change  it to executingCode, there's no need to add another field i think.